man/digest.Rd

```diff
@@ -1,9 +1,9 @@
 \name{digest}
 \alias{digest}
-\title{Create hash function digests for arbitrary R objects}
+\title{Create hash function digests for arbitrary R objects or files}
 \description{
-  The \code{digest} function applies a cryptographical hash function to
```

Comment:
Good catch to remove that.  Might be worth mentioning that the cryptography context is where the hash functions come from, but as it is now it is misleading.

Reply:
I will reword this in a way to say that one of many available hash functions is applied to a given object.  Not sure yet why, and as you point out these functions are both cryptographic and non-cryptographic.

Reply:
OK